### ISSUE TITLE
win11-vs2022-arm64-COMPILED_BY-vs2022-arm64 failure

### ISSUE THREAD

**Issue by ronag** [nodejs/build#3848]
Looks like there is something wrong with the machine?

https://ci.nodejs.org/job/node-test-binary-windows-native-suites/nodes=win11-vs2022-arm64-COMPILED_BY-vs2022-arm64/lastFailedBuild/console

```
13:52:39 not ok 142 node-api/test_general/test
13:52:39   ---
13:52:39   duration_ms: 918.90600
13:52:39   severity: fail
13:52:39   exitcode: 1
13:52:39   stack: |-
13:52:39     Can't clean tmpdir: d:\workspace\node-test-binary-windows-native-suites\node\test\.tmp.141
13:52:39     Files blocking: [ 'foo%#bar' ]
13:52:39     
13:52:39     node:fs:1246
13:52:39       return binding.rmSync(getValidatedPath(path), opts.maxRetries, opts.recursive, opts.retryDelay);
13:52:39                      ^
13:52:39     
13:52:39     Error: EPERM, Permission denied: \\?\d:\workspace\node-test-binary-windows-native-suites\node\test\.tmp.141\foo%#bar '\\?\d:\workspace\node-test-binary-windows-native-suites\node\test\.tmp.141\foo%#bar'
13:52:39         at Object.rmSync (node:fs:1246:18)
13:52:39         at Object.<anonymous> (d:\workspace\node-test-binary-windows-native-suites\node\test\node-api\test_general\test.js:34:6)
13:52:39         at Module._compile (node:internal/modules/cjs/loader:1546:14)
13:52:39         at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
13:52:39         at Module.load (node:internal/modules/cjs/loader:1317:32)
13:52:39         at Module._load (node:internal/modules/cjs/loader:1127:12)
13:52:39         at TracingChannel.traceSync (node:diagnostics_channel:315:14)
13:52:39         at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
13:52:39         at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:166:5)
13:52:39         at node:internal/main/run_main_module:30:49 {
13:52:39       errno: 1,
13:52:39       code: 'EPERM',
13:52:39       path: '\\\\?\\d:\\workspace\\node-test-binary-windows-native-suites\\node\\test\\.tmp.141\\foo%#bar',
13:52:39       syscall: 'rm'
13:52:39     }
13:52:39     
13:52:39     Node.js v23.0.0-pre
```

**Comment by targos** [comment]
@StefanStojanovic

**Comment by StefanStojanovic** [comment]
Thanks for taking it down @targos will investigate it a later during the day.

**Comment by StefanStojanovic** [comment]
Unfortunately, this is widespread, not machine-specific: and it started 1-2 days ago. The 3 tests affected by this are `parallel/test-child-process-fork-exec-path` in JS suites and `node-api/test_general/test` and `addons/load-long-path/test` from native suites. They all fail with `Can't clean tmpdir` because some file is blocking it. It is ARM64-specific. Continuing the investigation.

**Comment by RafaelGSS** [comment]
I believe this wasn't solved yet? https://github.com/nodejs/node/pull/54104 multiple runs with the same error

**Comment by StefanStojanovic** [comment]
> I believe this wasn't solved yet? [nodejs/node#54104](https://github.com/nodejs/node/pull/54104) multiple runs with the same error

I assume you are referring to the last 2 runs (from today). From what I see they were started via `Resume build`. The thing with that is that the compilation jobs are not rerun because they passed, thus the old compiled binaries are still used for testing. THe change that caused these problems was caused by compilation job changes, so besides tests, the compilation also needs to be rerun. It'd probably be best to rerun the entire `node-test-commit-windows-fanned` job.

**Comment by RafaelGSS** [comment]
Oh right. I have requested a new CI
